frontend/src/App.tsx

```diff
@@ -40,30 +70,48 @@ function App() {
             <header className="App-header">
                 <h1>Event Hub - Die digitale Litfaßsäule</h1>
                 {user !== undefined ? <button id="logout-button" onClick={logout}>Logout</button> : <></>}
-                <div id="nav-links">
-                    <div className="nav-link">{user !== undefined ? <Link to="/administration">Admin Area</Link> : <></>}</div>
-                    <div className="nav-link">{user !== undefined ? <Link to="/add">Organizer Area</Link> : <></>}</div>
-                    <div className="nav-link">{user !== undefined ? <Link to="/approve">Editor Area</Link> : <></>}</div>
-                </div>
+                {areNavLinksVisible && (
+                    <div id="nav-links">
+                        <div className="nav-link">{user !== undefined ?
```

Comment:
Why not use a wrapper div with the case validation which includes all those div?

Reply:
You mean the introduction of a separate component e.g. NavLinks that contains the visibility check together with the nav-links section?
If so: my plan was not to cast the navigation in the frontend into components yet, because I'm not quite sure how I want to implement the navigation in the end.